packages/core/strapi/src/core/loaders/plugins/get-enabled-plugins.ts

```diff
@@ -26,13 +26,33 @@ interface PluginDeclaration {
   isModule: boolean;
 }
 
-const INTERNAL_PLUGINS = [
-  '@strapi/plugin-content-manager',
-  '@strapi/plugin-content-type-builder',
-  '@strapi/plugin-email',
-  '@strapi/plugin-upload',
-  process.env.FEATURE_FLAG_CONTENT_RELEASES && '@strapi/content-releases',
-].filter(Boolean);
+/**
+ * This should be kept in sync with https://github.com/strapi/strapi/main/packages/core/admin/admin/src/render.ts#L10
```

Comment:
Not a huge fan of entirely not loading the plugin. I can already see long term we will want to make some feature flags manageable from the UI so users can more easily test new features. To make this work I would move this triggering the the Plugin init logic instead, so the plugin is still registered. 

wdyt ?

(Also avoid making changes in this part of the code that is going to be refactored in v5) 🙏

Reply:
A few things:

- Like all core plugins eventually we should remove them from the plugin list as they are not somethign you can install/uninstall. We use the same code architecture but we shouldn't consider them as plugins very long.
- We already show EE features in CE to show what users could use if they are interested so we could do the same for content-releases. Nothing new